### ISSUE TITLE
Service Account

### ISSUE THREAD

**Issue by wjohnsonsled** [EnCaseIntegratedToolkit/EITT#1]
Hi,

I am getting an error after I change the CySecModuleHostListener service account to run as Administrator instead of the Local Service account.  The error is "A call to SSPI failed, see inner exception."

Any ideas on this?

Thanks!

**Comment by markmorgan47** [comment]
Let me check with my enscriptor on this error and i will get back with you.

Mark Morgan

Las Vegas, NV. 89144

702-817 8796

**Comment by wjohnsonsled** [comment]
Hi,

I wanted to check in and see if you were able to have any luck with this
issue?

Thanks again!

**Comment by EnCaseIntegratedToolkit** [comment]
We are trying to reproduce the error.  Can you send me the log files.

**Comment by wjohnsonsled** [comment]
Yes which ones would you like?  I don't seem very many except for this one:  logfile = C:\Program Files (x86)\EnCase Integrated Threat Toolkit\CySecModuleHostListener.InstallLog

**Comment by markmorgan47** [comment]
can u do a webex with us.  Be easier that way.

On Friday, August 14, 2015, wjohnsonsled notifications@github.com wrote:

> Yes which ones would you like? I don't seem very many except for this one:
> logfile = C:\Program Files (x86)\EnCase Integrated Threat
> Toolkit\CySecModuleHostListener.InstallLog
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/EnCaseIntegratedToolkit/EITT/issues/1#issuecomment-131196821
> .

## 

Mark Morgan

Las Vegas, NV. 89144

702-817 8796

**Comment by wjohnsonsled** [comment]
Yes that is no problem.

**Comment by markmorgan47** [comment]
can you send me your email address.  This one only shows the github
notification address.  Once i get that i will send out an invite.

Mark Morgan

Las Vegas, NV. 89144

702-817 8796

**Comment by wjohnsonsled** [comment]
Yea just email me at w.walker.johnson@wellsfargo.com
